src/fstar/FStar.JsonHelper.fs

```diff
@@ -1,17 +1,23 @@
-(* Helpers mainly for FStar.Interactive.Lsp; some sharing with
- * FStar.Interactive.Ide *)
+(* Json helpers mainly for FStar.Interactive.Lsp; some sharing with *
```

Comment:
Can you please add the copyright notice to this and other new files? You can copy it from some other file that has it.

Reply:
Should I assign all copyrights to Nik?

Reply:
Certainly not! : ) Some files have copyright notices from when I first wrote them, so those mention my name; most do not. I've updated the style guide with a sample copyright notice. https://github.com/FStarLang/FStar/wiki/Style-guide

Reply:
For example, you could copy it from here: https://github.com/FStarLang/FStar/blob/master/src/fstar/FStar.CheckedFiles.fsi.